src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/record-jserror-exception.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: recordJSErrorException (Android SDK API)
+type: apiDoc
+shortDescription: Record JavaScript Error for ReactNative apps
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Android
+  - Android SDK API
+metaDescription: 'New Relic API for Android mobile app monitoring: record jserror for react native apps.'
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/setmaxeventpoolsize-android-agent-api
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/recordjserrorexception-android-sdk-api
+---
+
+## Syntax
+
+```
+NewRelic.recordJSErrorException(StackTrace stackTrace)
+```
+
+Record JavaScript Errors for ReactNative apps
+
+## Requirements
+
+Agent version 6.8.0 or higher, for hybrid apps
+
+## Description
+
+With this method, you can record JavaScript Errors from React Native apps.
+
+
+## Parameters
+
+This call accepts StackTrace parameter.
```

Comment:
@ywang-nr, I'd recommend adding backslashes to highlight this parameter like this: This call accepts the `StackTrace` parameter.